packages/ui-kit/src/components/AccessTokenProvider/index.ts

```diff
@@ -0,0 +1 @@
+export { AccessTokenProvider, AccessTokenProviderProps } from './AccessTokenProvider'
```

Comment:
Shouldn't we export `useAccessToken` here? I think customers should be able to import that. It will be useful for building custom components.

Reply:
I agree it might be useful, I'll export it